### ISSUE TITLE
ASCII error when importing a file with ñ in the name

### ISSUE THREAD

**Issue by castertr0y357** [evilhero/mylar#2245]
**Describe the bug**
When doing an import from an existing directory, Mylar hits a bug when it tries to import a file with ñ in the name.

**Screenshots/Logs**

2019-03-26 14:21:25 | ERROR | Uncaught exception: Traceback (most recent call last):File "/mylar/mylar/logger.py", line 337, in new_runold_run(*args, **kwargs)File "/usr/local/lib/python2.7/threading.py", line 754, in runself.__target(*self.__args, **self.__kwargs)File "/mylar/mylar/webserve.py", line 4230, in preSearchitself.addbyid(comicinfo['ComicID'], calledby=True, imported=imported, ogcname=comicinfo['ComicName'], nothread=True)File "/mylar/mylar/webserve.py", line 439, in addbyidreturn importer.addComictoDB(comicid, mismatch, None, imported, ogcname)File "/mylar/mylar/importer.py", line 414, in addComictoDBmoveit.movefiles(comicid, comlocation, imported)File "/mylar/mylar/moveit.py", line 45, in movefilesshutil.move(srcimp, dstimp)File "/usr/local/lib/python2.7/shutil.py", line 308, in moveos.rename(src, real_dst)UnicodeEncodeError: 'ascii' codec can't encode character u'\xf1' in position 68: ordinal not in range(128)
-- | -- | --
2019-03-26 14:21:25 | INFO | moving /media/comics/Extras/Spider-Man/Spidey06/Spin-Offs/Spider-Girl - Araña (Corazón)/Amazing Fantasy/Amazing Fantasy 001 (2004) (Digital) (AnPymGold-Empire).cbz ... to /media/Comics/Amazing Fantasy (2004)/Amazing Fantasy 1 (2004).cbz
2019-03-26 14:21:25 | DEBUG | Destination: /media/Comics/Amazing Fantasy (2004)/Amazing Fantasy 1 (2004).cbz
2019-03-26 14:21:25 | DEBUG | Source: Amazing Fantasy 001 (2004) (Digital) (AnPymGold-Empire).cbz
2019-03-26 14:21:25 | DEBUG | New Filename: Amazing Fantasy 1 (2004).cbz
2019-03-26 14:21:25 | DEBUG | new format: $Series $Issue ($Year)
2019-03-26 14:21:25 | DEBUG | not an annual - removing from filename paramaters
2019-03-26 14:21:25 | DEBUG | Comic Location: /media/Comics/Amazing Fantasy (2004)
2019-03-26 14:21:25 | DEBUG | Year: 2004
2019-03-26 14:21:25 | DEBUG | Series: Amazing Fantasy
2019-03-26 14:21:25 | DEBUG | Publisher: Marvel
2019-03-26 14:21:25 | DEBUG | Issue Year : 2004
2019-03-26 14:21:25 | DEBUG | Pretty Comic Issue is : 1
2019-03-26 14:21:25 | DEBUG | Zero level supplement set to none. Issue will be set as : 1
2019-03-26 14:21:25 | DEBUG | issue detected less than 10
2019-03-26 14:21:25 | DEBUG | Zero Suppression set to : none
2019-03-26 14:21:25 | DEBUG | issueid is now : 110252
2019-03-26 14:21:25 | DEBUG | annualize is None
2019-03-26 14:21:25 | DEBUG | issueid:None
2019-03-26 14:21:25 | DEBUG | issue# as per cv: 1
2019-03-26 14:21:25 | DEBUG | comicid: 18663
2019-03-26 14:21:25 | DEBUG |  
2019-03-26 14:21:25 | DEBUG |  
2019-03-26 14:21:25 | DEBUG | Renaming files according to configuration details : $Series $Issue ($Year)
2019-03-26 14:21:25 | DEBUG | original comicname is : Amazing Fantasy
2019-03-26 14:21:25 | DEBUG | comlocation is : /media/Comics/Amazing Fantasy (2004)
2019-03-26 14:21:25 | INFO | Mass import - Move files
2019-03-26 14:21:25 | INFO | Updating complete for: Amazing Fantasy
2019-03-26 14:21:25 | DEBUG | no update required - lastissueid [117225] = latestissueid [117225]
2019-03-26 14:21:24 | DEBUG | initiating issue updating - info & status
2019-03-26 14:21:24 | INFO | Now adding/updating issues for Amazing Fantasy

**Environment (please complete the following information):**
 - OS: Freenas Jail/ Freebsd 11.2-RELEASE-p6
 - Branch / commit: Development/eea5217 (latest as of 3/26/19)

**Additional information**
Running in freenas jail and cloned manually from git.

I would suspect this issue may extend to other series/issues with foreign characters.  If it's easier to fix it by simply changing the filename, let me know.

**Comment by evilhero** [comment]
It's actually not the filename, but the folder that would be causing some kind of problem (I tested against that particular series when working on the importer module).

However, I mocked up a fake directory & filename according to your logs, and was able to import without any issues:
```
INFO :: mylar.movefiles.47 : CP Server Thread-4 : moving /mnt/temp/tmp/Spider-Girl - Araña (Corazón)/Amazing Fantasy/Amazing Fantasy 001 (2004) (Digital) (AnPymGold-Empire).cbr ... to /mnt/Comics/Amazing Fantasy (2004)/Amazing Fantasy V2004 001 (2004).cbr
DEBUG :: mylar.movefiles.56 : CP Server Thread-4 : all files moved.
```

The only thing I didn't test it against was if the cbz included metadata and you had the metadata option enabled (cause for obvious reasons I can't mock that). Do you have the ```Use Existing Metadata``` option enabled  - and / or does the .cbz file contain metadata ? 

Although the more I think of it, It shouldn't matter, I mean the problem is with the folder name, not the file - albeit I can't duplicate the error on my Ubuntu, so I would think it has to be something localised on your machine that's causing the difference between our runs.

**Comment by castertr0y357** [comment]
I believe it does have metadata.  I picked this up as a giant download years ago and went through the arduous task of matching metadata within comicrack.  I am using existing metadata, but based on what you showed, it must be something local to my system, or in the freebsd code itself.

I ended up just changing the folder name, and I believe that took care of the problem.  Thanks for the assist.
